src/Duets.Cli/Components/Commands/Career/Work.Command.fs

```diff
@@ -25,11 +25,17 @@ module WorkCommand =
                     Effect.applyMultiple effects
 
                     Scene.WorldAfterMovement
-                | Error _ ->
+                | Error AttemptedToWorkDuringClosingTime ->
                     "The place is currently close, you can't work now!"
                     |> Styles.error
                     |> showMessage
 
+                    Scene.World
+                | Error AttemptedToWorkOnNonScheduledDay ->
+                    "You can't work today according to your schedule!"
```

Comment:
Change to: "Today is a free day for you. Try again on: {Formatted days}". Use one of the already available formatters to make it comma separated.